src/sentry/static/sentry/app/views/settings/incidentRules/create.tsx

```diff
@@ -35,21 +37,27 @@ class IncidentRulesCreate extends React.Component<Props> {
   };
 
   render() {
-    const {project, eventView, sessionId, ...props} = this.props;
+    const {project, eventView, sessionId, teams, ...props} = this.props;
     const defaultRule = eventView
       ? createRuleFromEventView(eventView)
       : createDefaultRule();
 
+    const userTeamIdArr = teams.filter(({isMember}) => isMember).map(({id}) => id);
+    const userTeamIds = new Set(userTeamIdArr);
+
+    defaultRule.owner = userTeamIdArr[0] && `team:${userTeamIdArr[0]}`;
```

Comment:
This is to calculate a default owner which will be pre-populated on the form on rule creation to encourage users to use the new ownership features